www/includes/easyparliament/templates/html/mp/votes_svg.php

```diff
@@ -1,45 +1,63 @@
 <?php
-    $all_lines = array();
+
+    $stances = array();
+
     foreach ($segment['votes']->positions as $key_vote) {
         if ( $key_vote['has_strong'] || $key_vote['position'] == 'has never voted on' ) {
-            $text = ucfirst(strip_tags($key_vote['desc']));
-        } else {
-            $text = "We don&rsquo;t have enough information to calculate $full_name &rsquo;s position on " . $key_vote['policy'] . ".";
```

Comment:
This was removed, but I don't see any code to ignore policies for which we would display on the site that we don't have enough information.